client/src/pages/ManageBookings/useStyles.ts

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    flexGrow: 1,
+    marginTop: '5rem',
+  },
+  bookingCard: {
+    marginBottom: '1rem',
+  },
+  booking: {
+    boxShadow: 'none',
+  },
+  cardHeader: {
+    padding: '1rem',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  bookingHeader: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  bookingTitle: {
+    fontWeight: 'bold',
+    textTransform: 'uppercase',
+  },
+  bookingAvatar: {
+    marginRight: '1rem',
+    marginTop: '1rem',
+    marginBottom: '1rem',
+  },
+  bookingProfileGrid: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },

```

Comment:
I don't see this being used in ManageBookings, if there are any unused styles here they should be removed.

Reply:
This is actually being used in Booking.tsx, on line 28.